sklearn/linear_model/_base.py

```diff
@@ -49,6 +50,61 @@
 # intercept oscillation.
 
 
+# FIXME in v1.2.0: variable 'normalize' should be removed from linear models
+# in cases where now normalize=False. default value of 'normalize' should be
+# changed to False in linear models where now normalize=True
+def _deprecate_normalize(normalize, default, est_name):
+    """
+    Normalize is to be deprecated from linear models and a use of a pipeline
+    with a StandardScaler is to be recommended instead.
+    Here the appropriate message is selected to be displayed to the user
+    depending on the default normalize value (as it varies between the linear
+    models and normalize value selected by the user). Est_name is the name of
+    the linear estimator which calls this function.
+
+    TODO: it should be updated in v1.2.0 and removed completely in v1.4.0
+    """
+    if normalize == 'deprecated':
+        _normalize = default
+    else:
+        _normalize = normalize
+
+    if default and normalize == 'deprecated':
+        warnings.warn(
+            " default of 'normalize' will be set to False in version 1.2.0 and"
```

Comment:
Could you remove the last `.0`. We don't use the patch version when warning usually, e.g. 1.0, 1.2, etc.